src/panels/lovelace/views/hui-view.ts

```diff
@@ -212,7 +263,24 @@ export class HUIView extends LitElement {
       });
     }
 
-    if (configChanged || editModeChanged || changedProperties.has("columns")) {
+    if (editModeChanged && !lovelace!.editMode) {
+      this._grids.forEach((grid) => {
+        grid.destroy();
+      });
+      this._grids = [];
+
+      if (!this._resizeObserver) {
+        return;
+      }
+      this._resizeObserver.disconnect();
+    }
+
+    if (
+      configChanged ||
+      editModeChanged ||
+      changedProperties.has("columns") ||
+      changedProperties.has("editCodeLoaded")
```

Comment:
TODO: Figure out the logic to not recreate the cards in edit mode when cards are moved